api/main.go

```diff
@@ -14,6 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+//go:generate statik -src=./dashboard -f
```

Comment:
It means it is optional, right?

Reply:
It describes the instructions that will be executed when you run `go generate` directly. Users can use the static command instead of go generate.
This command does not affect the original compilation process.